bookwyrm/templatetags/utilities.py

```diff
@@ -125,7 +125,8 @@ def id_to_username(user_id):
         name = parts[-1]
         value = f"{name}@{domain}"
 
-    return value
+        return value
+    return "a new user account"
```

Comment:
I'm going to merge this since it's an active bug, but this should be localized as `_(a new user account)`